frontend/packages/dev-console/integration-tests/features/addFlow/developer-Catalog-Details.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Developer Catalog Page
+    As a user, I should be able to use Developer Catalog page to deploy application
+
+
+    Background:
+        Given user is at developer perspective
+        And user has installed OpenShift Serverless Operator
+        And user has installed Knative Apache Camelk Operator
+        And user has installed Apache Kafka Operator
```

Comment:
Why does user needs to install all these operators for using developer catalog? The catalog is available for all users without any operators installed.

Reply:
Updated.